types/auth0-js/index.d.ts

```diff
@@ -331,12 +336,6 @@ export class Redirect {
 export class Popup {
     constructor(client: any, options: any);
 
-    /**
-     * Returns a new instance of the popup handler
-     *
-     */
-    buildPopupHandler(): any;
```

Comment:
I recommend marking this as `private` instead of fully removing it - that way, if consumers are calling it, they will get a more helpful error message.

Reply:
Done, rebased, squahsed and commit message changed